### ISSUE TITLE
NTR: Defective interstrand cross-link repair

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#3562]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Defective interstrand cross-link repair

### 2. Synonyms
N/A

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
A defect in the of the process of interstrand cross-link repair: removal of a DNA interstrand crosslink (a covalent attachment of DNA bases on opposite strands of the DNA) and restoration of the DNA. DNA interstrand crosslinks occur when both strands of duplex DNA are covalently tethered together (e.g. by an exogenous or endogenous agent), thus preventing the strand unwinding necessary for essential DNA functions such as transcription and replication.
(def modified from GO: GO_0036297, PMID:16464006, PMID:22064477)

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0003254 'Abnormality of DNA repair'

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Fanconi Anemia

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
https://orcid.org/0000-0001-5208-3432

### 7. Need to MIREOT:
GO_0036297

**Comment by pnrobinson** [comment]
This term as well as the annotation look good to me. We will need to annotate all of the Fanconis probably.

**Comment by nicolevasilevsky** [comment]
@drseb could you MIREOT GO_0036297, please?

**Comment by pnrobinson** [comment]
@nicolevasilevsky  Returning to this, I am uncertain if this is a phenotypic abnormality that can be measured in the clinic -- it seems like it is more of an inference about the mechanism that is an experimental finding. The other DNA repair terms are things that I believe occasionally get measured clinically. I think that this needs to go somewhere else! I will close this issue, but if there actually is a clinical measurement that has this as its results we can reconsider.

**Comment by nicolevasilevsky** [comment]
@pnrobinson Maureen said: The diagnostic criteria for FA is a cellular hypersensitivity to MMC and/or DEB (chemicals that induce DNA damage), measurements that are performed clinically.

Should we obsolete this class and create two new classes: cellular hypersensitivity to MMC and cellular hypersensitivity to DEB?

**Comment by pnrobinson** [comment]
We should definitley have terms for the measurements that are performed clinically. I do not think that this NTR has actually been added yet?

https://hpo.jax.org/app/browse/term/HP:0003254

**Comment by pnrobinson** [comment]
The other two terms have been added in separate NTRs. We can close this one!
